File: 1460. Make Two Arrays Equal by Reversing Subarrays.php
class Solution {
    function canBeEqual($target, $arr) {
    sort($target);
    sort($arr);
    for ($i = 0; $i < count($arr); $i++) {
        if ($arr[$i] !== $target[$i]) {
            return false;
        }
    }
    return true;
    }
}

File: 1460. Make Two Arrays Equal by Reversing Subarrays.rs
impl Solution {
    pub fn can_be_equal(mut target: Vec<i32>, mut arr: Vec<i32>) -> bool {
        target.sort();
        arr.sort();
        for i in 0..arr.len() {
            if arr[i] != target[i] {
                return false;
            }
        }
        true  
        }
} ////Amazing Rust, I love it
